llama-swap-windows.go

```diff
@@ -0,0 +1,100 @@
+//go:build windows
+
+package main
+
+import (
+	_ "embed"
+	"flag"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/getlantern/systray"
+	"golang.org/x/sys/windows"
+)
+
+const TargetURL = "http://localhost"
+
+//go:embed ui/public/favicon.ico
+var iconData []byte
+var tray *bool
+
+func addFlagsIfNeed(flag *flag.FlagSet) {
+	tray = flag.Bool("tray", false, "add tray icon")
+}
+
+func restartIfNeed() {
+	if !*tray {
+		return
+	}
+
+	kernel32 := syscall.MustLoadDLL("kernel32.dll")
+	getConsoleWindow := kernel32.MustFindProc("GetConsoleWindow")
+	ret, _, _ := getConsoleWindow.Call()
+	if ret == 0 {
+		return
+	}
+
+	programPath := os.Args[0]
+	cmd := exec.Command(programPath, os.Args[1:]...)
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		HideWindow:    true,
+		CreationFlags: windows.CREATE_NO_WINDOW,
+	}
+	err := cmd.Start()
+	if err != nil {
+		log.Fatalf("Fatal restart error: %v\n", err)
+	}
+	os.Exit(0)
+}
+
+func runTrayIfAvailable() {
+	if !*tray {
+		return
+	}
+
+	systray.Run(onReady, onExit)
+}
+
+func onReady() {
+
+	systray.SetIcon(iconData)
+	systray.SetTitle("llamaSwap")
+	systray.SetTooltip("llamaSwap")
+
+	mOpenLog := systray.AddMenuItem("Open web ui", "Open llamaSwap-Web front page")
```

Comment:
I think We only need two options: 

- Show UI, "open user interface in default browser"
- Quit, "quit llama-swap process"

I'm OK with keep the link to the model page but it seems strange to have both when they go to essentially the same place.